docs/reference/contributing/guidelines/workflow.md

```diff
@@ -79,6 +79,43 @@ Each feature has a tech lead. This person is responsible for:
 - Rebasing often to track master development.
 - Reviewing any addition to the feature branch (approval required by the feature tech lead or another assigned person).
 
+##### Pull request types
+
+We consider the following pull request types.
+
+##### Fix
+
+A bug fix should be backward-compatible internal change that fixes incorrect behavior. The fix should provide a description what and how it fixes the issue. The test should be included to catch this issue in the future and confirm the correct behavior.
+
+Release: patch version
```

Comment:
Do we go over versions somewhere in docs? maybe we should link there in this page so these terms are defined? @AnotherButler?

Reply:
:+1: